src/Installers/Windows/AspNetCoreModule-Setup/IIS-Setup/include.wxi

```diff
@@ -23,6 +23,12 @@
     <?define ProgramFilesFolder = ProgramFilesFolder ?>
     <?define SystemFolder = SystemFolder ?>
     <?define PlatformValue = Intel ?>
+  <?elseif $(var.Platform) ~= "arm64" ?>
```

Comment:
Why `~=` here as opposed to `=` above?

Reply:
I think the difference is only case insensitive, I can switch it to equals to be consistent if you prefer, we seem to be inconsistent with ARM64/arm64 more than the other platforms though